### PR TITLE
Update existing email address

### PR DESCRIPTION
**Pull request by ArondeParon**
It currently is not possible to directly update the email address of an existing subscriber without using `getApi()` directly. 

This pull requests implements a simple method to do exactly this.

### CODE REVIEW

**Comment by freekmurze** [comment]
Nice, thanks! Could you also update the readme with an example and add at test?

**Comment by ArondeParon** [comment]
Sure thing, will do!

**Comment by ArondeParon** [comment]
I'll look into later tonight or this weekend. Thought I could simply do this in a couple of minutes through Github since I'm not at my laptop right now, but it seems there's a bit more to it than I thought :)

File: tests/MailChimp/NewsletterTest.php [hunk 1]
```diff
@@ -240,6 +240,33 @@ public function it_can_override_the_defaults_when_subscribing_or_updating_someon
 
         $this->newsletter->subscribeOrUpdate($email, [], '', ['email_type' => 'text', 'status' => 'pending']);
     }
+    
+    /** @test */
+    public function it_can_change_the_email_address_of_a_subscriber()
+    {
+        $email = 'freek@spatie.be';
+        $newEmail = 'phreak@spatie.be';
```
Comment (freekmurze) [inline tests/MailChimp/NewsletterTest.php]: haha 😄

Comment (ArondeParon) [inline tests/MailChimp/NewsletterTest.php]: 🙈

**Comment by ArondeParon** [comment]
Tests passing now! 👍

**Comment by freekmurze** [comment]
Awesome PR, thanks! 👍

**Pull request merged**